src/lib/data/data_types.ts

```diff
@@ -4,10 +4,11 @@ interface Comments {
   sga_treasurer_comment?: string;
 }
 // an Item will represent a single item in a budget
-interface Item extends Comments {
+interface Item extends Comments, Document {
+  budget_id: string;
   name: string;
   quantity: number;
-  unit_price: number;
+  cost: number;
```

Comment:
Question: is there a reason for the change from `unit_price` to `cost`? I feel that `cost` might be a bit confusing for items where `quantity > 1`. I think a name that reflects either the cost being for *one* item  or for many items might help us avoid mistakes